Bio/PDB/internal_coords.py

```diff
@@ -4207,7 +4207,7 @@ def __init__(
                     akl.append(fld)
 
         # tweak local akl to generate id string
-        if isinstance(akl[0], int):
+        if isinstance(akl[0], (int, np.integer)):
```

Comment:
You can use `isinstance(akl[0], numbers.Integral)` for this.

Reply:
Thanks @mdehoon, I'm not otherwise importing numbers but would that be preferable?